### PR TITLE
test: adding debug output in ITPaginationTest

### PR DESCRIPTION
**Pull request by suztomo**
Troubleshooting for the flaky test https://github.com/googleapis/java-compute/issues/683.

### CODE REVIEW

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java [hunk 1]
```diff
@@ -114,13 +114,18 @@ public void testPaginationIterating() {
         ListZonesRequest.newBuilder().setProject(DEFAULT_PROJECT).setMaxResults(1).build();
     ZonesClient.ListPagedResponse response = zonesClient.list(listZonesRequest);
     boolean presented = false;
+    int count = 0;
```
Comment (mpeddada1) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:117]: optional: Perhaps we could name this `numZones`?

**Review by mpeddada1 — APPROVED** [review]

**Review by meltsufin — APPROVED** [review]
Can you debug locally?

**Comment by suztomo** [comment]
@meltsufin No, I don't have local setup.

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java (hunk 1 above)
Comment (suztomo) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:117]: I go for `count` for shorter name.

**Pull request merged**